app/src/main/java/com/github/polybooks/database/FBBookDatabase.kt

```diff
@@ -112,6 +111,18 @@ object FBBookDatabase: BookDatabase {
         )
     }
 
+    private fun getBooksByISBNFromFirebase(isbns : List<ISBN>) : CompletableFuture<List<Book>> {
```

Comment:
Isn't that just the `getBooks()` function ?

Reply:
J0'en sais rien frérot

Reply:
c'est un truc que j'ai du merge conflict
aucune idée
je sais pas